### PR TITLE
Throw an appropriate error when a table for which View/Edit data is open is deleted and query is executed

### PR DESCRIPTION
**Pull request by nikhil-mohite**

### CODE REVIEW

File: web/pgadmin/tools/sqleditor/__init__.py [hunk 1]
```diff
@@ -761,6 +762,22 @@ def start_view_data(trans_id):
 
     if status and conn is not None and \
             trans_obj is not None and session_obj is not None:
+
+        # Fetch the Namespace Name and object Name
+        sql_path = 'sqleditor/sql/#{0}#'.format(manager.version)
```
Comment (adityatoshniwal) [inline web/pgadmin/tools/sqleditor/__init__.py:767]: Wouldn't the query execute give an error? Like table does not exist?

**Review by adityatoshniwal — CHANGES_REQUESTED** [review]

File: web/pgadmin/tools/sqleditor/__init__.py (hunk 1 above)
Comment (nikhil-mohite) [reply web/pgadmin/tools/sqleditor/__init__.py:767]: We fixed this issue earlier but the fix was not right #6138.

**Comment by khushboovashi** [comment]
This PR will also fix #6354 too.

File: web/pgadmin/tools/sqleditor/__init__.py (hunk 1 above)
Comment (adityatoshniwal) [reply web/pgadmin/tools/sqleditor/__init__.py:767]: I mean, if the object does not exist in the database then query tool itself will give error in the messages tab?

**Pull request closed without merging**